cmake/ruyConfig.cmake.in

```diff
@@ -0,0 +1,7 @@
+include(CMakeFindDependencyMacro)
```

Comment:
Can you please add a comment at the start of this file explaining its purpose? How do you plan to consume this file?

Reply:
I have added the comment, this follows the CMake conventions for packages and will be picked up automatically by `find_package(ruy)`. See https://github.com/abseil/abseil-cpp/blob/1963f10ae5cb32a7ea6d96b928f69d3c7fba0139/CMake/abslConfig.cmake.in for an example of an equivalent file in Abseil.

Reply:
Thanks!